snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -234,6 +236,47 @@ def _wait_for_network(self):
                         'No network connection in the container.\n'
                         'If using a proxy, check its configuration.')
         logger.info('Network connection established')
+        self._setup_vendoring()
+
+    def _setup_vendoring(self):
+        vendoring = self._project_options.info.vendoring
+        if vendoring:
+            logger.info(
+                'Vendoring is set to {}. Access to external resources not '
+                'specified here will be denied during the build.'
+                .format(', '.join(vendoring)))
```

Comment:
This message gets nasty quickly if there are several items in the vendoring list. It is, however, helpful to show them here, so I suggest a reformat. Perhaps:

"This snap uses vendoring. In order to enforce it, network access to resources not on the vendor list will be denied. The vendor list is:\n{}" and print each domain on its own line. Something like that, anyway. I also wonder if this should be a warning given that denials are so very opaque.